elasticsearch/file/src/main/java/nl/knaw/huc/exception/TextRepoRequestException.java

```diff
@@ -0,0 +1,12 @@
+package nl.knaw.huc.exception;
+
+public class TextRepoRequestException extends IllegalArgumentException {
```

Comment:
Dit is het punt waarop een IllegalArgumentException een TextRepoRequestException wordt, dat wordt later een 503 (zie de Mapper).
IllegalArgument voelt meer als 400 als het vanuit een endpoint komt waar het een reactie is op een verkeerde user parameter.

Reply:
Ah ja, deze IllegalArgumentExceptions kunnen een BadRequestException worden

Reply:
812b1022ff135c01835adacc1ff311c7828a2c7a